crates/node_binding/src/js_values/compilation.rs

```diff
@@ -383,6 +385,32 @@ impl JsCompilation {
       .build_dependencies
       .extend(deps.into_iter().map(PathBuf::from))
   }
+
+  #[napi]
+  pub fn rebuild_module(
+    &'static mut self,
```

Comment:
@h-a-n-a I'm not sure whether this `static` has unexpected effects, please help to review carefully